vapi.ts

```diff
@@ -113,20 +112,27 @@ export default class Vapi extends VapiEventEmitter {
   }
 
   async start(
-    assistant: CreateAssistantDTO | string,
-    assistantOverrides?: OverrideAssistantDTO,
+    assistant?: CreateAssistantDTO | string,
```

Comment:
given breaking change, could just make it the full CreateWebCallDTO object

Reply:
reverted w squads